### ISSUE TITLE
Date and time format

### ISSUE THREAD

**Issue by skycommand** [aleksey-hoffman/sigma-file-manager#188]
### Is there an existing issue for this?

- [X] I have searched the opened issues

### Problem description

The app doesn't respect regional settings and the user's preferences regarding the date and time formats. Instead, it uses the following settings:

- For the date format, it uses the month-first format, which only 18% of people in the world use, mainly in the U.S. and Japan.
- For the time format, the app uses the 12-hours format, instead of the more concise 24-hours format.

### Steps to reproduce

Just open the app and look at any date field.

### Environment

- OS: Windows 10 v21H2
- App version: 1.5.0

### Notes

Since this is a Windows app, you can have Windows format the date and time for you. Depending on which API you are using, it could be as easy as this: 
```csharp
return DateTime.Now.ToString("D"); //In my computer, returns 18 Aug 2022, Thu
```
The `ToString` function accepts "F", "f", "D", "d", "G", and "g".

Here are two sets of documentation, in case you need them: [.NET](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings) and [Windows API](https://docs.microsoft.com/en-us/windows/win32/intl/retrieving-time-and-date-information)

**Comment by aleksey-hoffman** [comment]
Hi, thanks for the feedback
It's already supposed to use your system default locale to format date and time.
It seems your system also has `en-US` locale installed and Sigma uses that as the default.

If you'd like to help me debug this so I can fix it, please do the following:

In your browser, right on this page, press `Ctrl + Shift + I` then go to `Console` tab and then run the following line of code there: `window.navigator.languages`

Then please show me the result. You should see something like this:

![image](https://user-images.githubusercontent.com/61761672/185445742-3c3e934d-0750-4e75-aed4-6129574e5120.png)

**Comment by skycommand** [comment]
> It's already supposed to use your system default locale to format date and time.
> It seems your system also has en-US locale installed and Sigma uses that as the default.

No, it doesn't. Here is a screenshot.

![My regional settings](https://user-images.githubusercontent.com/17097175/185450346-e4e45958-394a-44f0-b606-0545aa32e2a5.png)

And in PowerShell, I get this:

```
PS C:\> [System.Threading.Thread]::CurrentThread.CurrentCulture

LCID             Name             DisplayName
----             ----             -----------
4096             en-001           English (World)
```

> In your browser, right on this page, press `Ctrl + Shift + I` then go to `Console tab` and then run the following line of code there: `window.navigator.languages`

That's a strange request, but I oblige. Here:

![My browser's regional settings](https://user-images.githubusercontent.com/17097175/185452359-2657abfe-6b21-48cd-a730-461873ac6d70.png)

I asked my colleagues about this request. Browsers do not adhere to OS regional settings for privacy and customization reasons. (The only exception is Internet Explorer.) I can go to Firefox or Microsoft Edge's options and change the outcome of this JavaScript command.

**Comment by aleksey-hoffman** [comment]
Thanks, mate. I found out that the function that I'm using formats `en` locale as `en-US`, rather than `en-GB`
I'm gonna add an option to specify any locale format, in the next update

**Comment by aleksey-hoffman** [comment]
Fixed in 89eaa9a
Will be released in v1.6 update

![image](https://user-images.githubusercontent.com/61761672/191084945-56300327-9bff-4010-9558-e847ba0f4e56.png)
